Developers/Writing_Good_Commits.rst

```diff
@@ -7,9 +7,7 @@ At coala we are looking heavily at the maintainability of the code.
 
     Code is more often read than written!
 
-It is obvious that we need good code. In order to do that we are
-verifying that every change to our code (i.e. the commits) is making it
-better.
```

Comment:
`....good code <new-line>` . Try to see at what column is the last character, in your editor and wrap it if needs